supportedBackends/tes/src/main/scala/cromwell/backend/impl/tes/TesRuntimeAttributes.scala

```diff
@@ -151,6 +151,26 @@ object PreemptibleValidation {
 class PreemptibleValidation extends BooleanRuntimeAttributesValidation(TesRuntimeAttributes.PreemptibleKey) {
   override def usedInCallCaching: Boolean = false
 
+  override protected def validateExpression: PartialFunction[WomValue, Boolean] = {
+    case womBoolValue if womType.coerceRawValue(womBoolValue).isSuccess => true
```

Comment:
Is it possible to compose partial functions so we can re-use the parent implementation? (This is a gap in my Scala knowledge.)

Reply:
I'm not 100% sure, but I don't think so. You can compose partial functions as in using the output of one as the input of another, but that's not what we want here - if we called the parent function and the result was `false` because the `WomValue` couldn't be converted to boolean, we'd no longer have the initial input in hand to attempt to convert to int.